.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
+yarn.lock
```

Comment:
```suggestion
node_modules
```

Instead of adding lock files to the gitignore I think it's better to add an `.npmrc` file with `package-lock=false` in it. Otherwise people will be using a lock-file locally, it just won't be synchronised...

Reply:
I think a better solution would be to agree to use just one package manager to develop this and add it to a `contributions` or `requirements` section of the readme.

P.D: this PR just turned 1 year old 🥳